src/TagsInput/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import styled from 'styled-components';
+import Input, { InputProps } from '../Input';
+import Tag from '../Tag';
+
+const StyledTagsInput = styled.ul`
+  background: white;
+  box-sizing: border-box;
+  border: 1px solid #d2d6dc;
+  padding: 5px 32px 5px 12px;
+  border-width: 1px;
+  border-radius: 4px;
+  transition: all 250ms ease 0s;
+  outline: none;
+  width: 100%;
+  display: inline-flex;
+  flex-wrap: wrap;
+
+  li {
+    margin: 0 2px;
+  }
+`;
+
+const StyledInput = styled(Input)`
+  border: none;
+
+  &:focus {
+    box-shadow: none;
+  }
+`;
+
+export interface TagsInput extends InputProps {
+  tags: string[];
+  onClickIcon: (index: number) => void;
+}
+
+export const TagsInput = function TagsInput({
+  tags,
+  onClickIcon,
+  ...props
+}: TagsInput) {
+  return (
+    <StyledTagsInput>
+      {tags &&
+        tags.map((tag, i) => (
+          <li key={tag + i}>
+            <Tag
+              data-testid="tag"
+              showIcon={true}
+              onClickIcon={e => {
+                e.preventDefault();
+                onClickIcon(i);
+              }}
+            >
+              {tag}
+            </Tag>
+          </li>
+        ))}
+      <li>
+        <StyledInput data-testid="input" {...props} />
```

Comment:
It looks like this styled input isn't the full width of the input field, which means if you click on the right side here it won't focus the input:

![image](https://user-images.githubusercontent.com/22530815/77104020-530ed980-69e9-11ea-9042-d5d636814dda.png)

I would recommend reworking this slightly to make the entire area clickable. It might require you to use a ref to force the focus on the main input?